.github/workflows/test.yml

```diff
@@ -46,6 +46,13 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      
+      - name: Install previous major version for testing upgrade
```

Comment:
I think you should `Get Compose` first.

Also, if we are doing this some of this should become obsolete:
https://github.com/getsentry/self-hosted/blob/013519a46c696438016a1577615d47f1ec677212/integration-test.sh#L23-L26